SConstruct

```diff
@@ -320,6 +320,7 @@ env.OverrideFromArg('STRIP', default='strip')
 env.OverrideFromArg('DOXYGEN', default='doxygen')
 env.OverrideFromArg('SPHINX_BUILD', default='sphinx-build')
 env.OverrideFromArg('BREATHE_APIDOC', default='breathe-apidoc')
+env.OverrideFromArg('PKG_CONFIG', default='pkg-config')
```

Comment:
We already has this call for PKG_CONFIG here:

https://github.com/roc-project/roc/blob/05b998cce82fcc4e6ba9de96ab68821075747606/SConstruct#L312-L313

And we should add PKG_CONFIG_PATH.

BTW if PKG_CONFIG_PATH wont have a default, we can just add it to that list at line 312.

Reply:
We also should add PKG_CONFIG_PATH to this list: https://roc-project.github.io/roc/docs/building/scons_options.html#variables